JPS_Ontology/ontology/ontoems/OntoEMS.owl

```diff
@@ -657,7 +730,6 @@
 
 
     <owl:Class rdf:about="https://www.theworldavatar.com/kg/ontoems/AirTemperature">
-        <owl:equivalentClass rdf:resource="https://www.auto.tuwien.ac.at/downloads/thinkhome/ontology/WeatherOntology.owl#Temperature"/>
```

Comment:
Why are you removing all those equivalent class definitions?

Reply:
Removal of equivalent class definitions and OM prefix were unintential changes by TBoxGeneration, have been reverted manually.